debug/index.html

```diff
@@ -58,7 +58,8 @@
     zoom: 12.5,
     center: [-77.01866, 38.888],
     style: 'mapbox://styles/mapbox/streets-v8',
-    hash: true
+    hash: true,
+    workerCount: 3
```

Comment:
@mourner My thinking was that, if the majority of users have 4 cores, it might actually be beneficial to have developers working under those conditions by default.